### PR TITLE
SC-15298 Support Alpine 3.18

### PR DESCRIPTION
**Pull request by kalaputsik**
### Description

- Ticket/Issue: https://spryker.atlassian.net/browse/SC-15041

#### Related resources

- https://github.com/spryker/docker-sdk/pull/463

#### Change log

- Introduced images with PHP 8.1 and 8.2 versions based on Alpine 3.18
- Upgraded Newrelic version on the latest
- Updated Composer version to the latest.
- Updated PHP versions to the latest. 

### Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

**Review by zyuzka — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -35,7 +35,15 @@ jobs:
...
-            tags: [ "spryker/php:latest", "spryker/php:8.2", "spryker/php:8.2-alpine3.17" ]
+            tags: [ "spryker/php:8.2", "spryker/php:8.2-alpine3.17" ]
+            platforms: [ "linux/amd64", "linux/arm64" ]
+
+          ### Alpine 3.18
+          - image: "alpine/3.18/8.1/Dockerfile"
+            tags: [ "spryker/php:8.1", "spryker/php:8.1-alpine3.18" ]
+            platforms: [ "linux/amd64", "linux/arm64" ]
+          - image: "alpine/3.18/8.2/Dockerfile"
+            tags: [ "spryker/php:latest", "spryker/php:8.2", "spryker/php:8.2-alpine3.18" ]
```
Comment (alexanderM91) [inline .github/workflows/ci.yml:46]: Duplicated tag - spryker/php:8.2

File: .gitignore [hunk 2]
```diff
@@ -1 +1,2 @@
 .idea
+utils.sh
```
Comment (alexanderM91) [inline .gitignore:2]: Do we need it?

File: alpine/3.15/8.0/Dockerfile [hunk 3]
```diff
@@ -1,16 +1,13 @@
 # syntax = docker/dockerfile:1.0.2-experimental
 ARG SPRYKER_PHP_VERSION=8.0.26
```
Comment (alexanderM91) [inline alpine/3.15/8.0/Dockerfile:2]: Could you please recheck the PHP version and the composer? 
Thanks.

File: .gitignore (hunk 2 above)
Comment (zyuzka) [reply .gitignore:2]: For my script

File: alpine/3.15/8.0/Dockerfile (hunk 3 above)
Comment (zyuzka) [reply alpine/3.15/8.0/Dockerfile:2]: Updated

File: .github/workflows/ci.yml (hunk 1 above)
Comment (zyuzka) [reply .github/workflows/ci.yml:46]: Updated

**Review by alexanderM91 — APPROVED** [review]

**Pull request merged**